cve_bin_tool/checkers/README.md

```diff
@@ -317,18 +317,18 @@ You can set an arbitrary number of workers. A good rule of thumb is to specify n
 
 The CVE-bin-checker works by extracting strings from binaries and determining
 if a given library has been compiled into the binary. For this, Checker class
-contains two methods: 1) guess_contains and 2) get_version.
+contains two methods: 1) `guess_contains()` and 2) `get_version()`.
 
-1. guess_contains method takes list of extracted string lines as an input and 
+1. `guess_contains()` method takes list of extracted string lines as an input and 
 return True if it finds any of the `CONTAINS_PATTERNS` on any line from the 
 lines.
-2. get_version method takes list of extracted string lines and the filename as
+2. `get_version()` method takes list of extracted string lines and the filename as
 inputs and returns information about whether the binary contains the library
 in question, is a copy of the library in question, and if either of those are
 true it also returns a version string. If the binary does not contain the
 library, this function returns an empty dictionary.
 
-If `curl` product is being scanned, get_version method of CurlChecker will
+If `curl` product is being scanned, `get_version()` method of CurlChecker will
 return following dictionary.
```

Comment:
Changed get_version -> `get_version()` and guess_contains -> `guess_contains()` as they were written in this style in Blockquotes above. Also, they were functions, which were written in a similar fashion in the document.